script/utils/bilibili_spider.py

```diff
@@ -0,0 +1,195 @@
+import re

```

Comment:
Please specify the origin bilibili_spider URL, we need to keep the license information of this file.